.github/dependabot.yml

```
@@ -120,6 +120,10 @@ updates:
       # Micrometer
       - dependency-name: io.micrometer:micrometer-bom
       - dependency-name: io.micrometer:micrometer-registry-stackdriver
+      # BouncyCastle
+      - dependency-name: org.bouncycastle:bcprov-jdk15on
```

Maybe it's easier to just do:
```suggestion
      - dependency-name: org.bouncycastle:*
```

WDYT?

---

Hi @gastaldi I think we had some problems with a wildcard for some group id with a similar situation, here we `bc-fips` is on a different version `1.0.2` to the other 2 deps (`1.66`) ? I recall we discussed something similar with Guillaume, CC @gsmet

---

Ah okay, no problem then 👍